lib/IRGen/GenDecl.cpp

```diff
@@ -4715,7 +4719,8 @@ llvm::Constant *IRGenModule::getAddrOfGlobalString(StringRef data,
     return entry.second;
   }
 
-  entry = createStringConstant(data, willBeRelativelyAddressed);
+  entry = createStringConstant(data, willBeRelativelyAddressed,
+                               "" /*no section*/, ".str");
```

Comment:
The general style is that the comment should precede the parameter for the annotation.